src/components/EditProfile.js

```diff
@@ -0,0 +1,87 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+
+class EditProfile extends Component {
```

Comment:
Suggestion: We could just make one component that is both show and edit view for the user's profile.  Then we can just pass in a different user and reuse the same component if we want to allow the user to see other people's profile pages. :)